### PR TITLE
Second round of style conventions edits for: "Deploy a token bridge using the Orbit SDK"

### PR DESCRIPTION
**Pull request by anegg0**
Goal of this PR: getting content aligned with OCLC's [style guide](https://www.notion.so/arbitrum/Pattern-guide-Docs-dafaf8d1ce34433d88c1ef3b4b99c19c?pvs=4#b420b2b1ff614c71b01743e1b140ffbe)

Currently being reviewed by @symbolpunk

### CODE REVIEW

File: arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-token-bridge.md [hunk 1]
```diff
@@ -135,25 +166,24 @@ const tokenBridgeContracts = await txReceipt.getTokenBridgeContracts({
...
+That step only applies to `ETH`-based Orbit chains, not Custom fee token orbit chains. Our canonical bridge design has a separate custom gateway for `WETH` to bridge it in and out of the Orbit chain. 
 
-You can find more info about WETH gateways in our ["other gateways flavors" documentation](https://docs.arbitrum.io/for-devs/concepts/token-bridge/token-bridge-erc20#other-flavors-of-gateways).
+You can find more info about `WETH` gateways in our ["other gateways flavors" documentation](https://docs.arbitrum.io/for-devs/concepts/token-bridge/token-bridge-erc20#other-flavors-of-gateways).
 
 :::
 
-So, after the token bridge has been deployed and you have secured a successful deployment on both parent and child chains, it's time to set the `WETH Gateway` on both parent and child chains. To handle that, we have two APIs on our Orbit SDK:
+So, after the token bridge has been deployed and you have secured a successful deployment on both parent and child chains, it's time to set the ``WETH` Gateway` on both parent and child chains. To handle that, we have two APIs on our Orbit SDK:
```
Comment (GreatSoshiant) [inline arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-token-bridge.md:181]:
```suggestion
So, after the token bridge has been deployed and you have secured a successful deployment on both parent and child chains, it's time to set the `WETH` Gateway on both parent and child chains. To handle that, we have two APIs on our Orbit SDK:
```

File: arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-token-bridge.md [hunk 2]
```diff
@@ -169,25 +199,31 @@ const setWethGatewayTxRequest = await createTokenBridgePrepareSetWethGatewayTran
...
+
+After creating the raw transaction, you need to use Viem to sign it and broadcast it to the network.
 
 #### 2. `createTokenBridgePrepareSetWethGatewayTransactionReceipt`
 
-After sending the transaction, you need get the recept of the transaction to be able to check about the success of the Retryable Tickets created on step 1, which is going to set WETH gateway on the Orbit chain. To do that we are using `createTokenBridgePrepareSetWethGatewayTransactionReceipt` API and also `waitForRetryables` method of it to check for the retryable ticket status. For the example in this doc we can use this API as follow:
+After sending the transaction, you need to assess if the Retryable Tickets you just created have been successful.. To do that we are using `createTokenBridgePrepareSetWethGatewayTransactionReceipt` API and the`waitForRetryables` method of it to check for the success status of retryable tickets. For the example in this doc we can use this API as follow:
```
Comment (GreatSoshiant) [inline arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-token-bridge.md:208]:
```suggestion
After sending the transaction, you need to assess if the Retryable Tickets you just created have been successful. To do that we are using `createTokenBridgePrepareSetWethGatewayTransactionReceipt` API and the`waitForRetryables` method of it to check for the success status of retryable tickets. For the example in this doc we can use this API as follow:
```

File: arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-token-bridge.md [hunk 3]
```diff
@@ -24,30 +26,36 @@ See the [`ERC-20` token bridge overview](/build-decentralized-apps/token-bridgin
...
 
-The token bridge deployment process depends on the type of Orbit chain. In the following steps, the main flow is the same for all different kinds of Orbit chains except [step 1](#step-1), which is only needed for **Custom fee token** Orbit chains, and [step 5](#step-5), just for `ETH`-based Orbit chains.
+The token bridge deployment process is the same for all Orbit chains types except for the following:
+- **Custom fee token Orbit chains** which require [token approval](#step-1).
+- **`ETH`-based Orbit chains**, for which you need to [set up a WETH gateway](#step-5).
 
 :::
 
-### 1. Token Approval (Custom fee token Orbit chains only){#step-1}
+### 1. Token approval {#step-1}
```
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-token-bridge.md:47]: Curious - is there any reason we can't remove these unconventional `{}` notations and just link to the headers as-usual, where needed?

File: arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-token-bridge.md [hunk 4]
```diff
@@ -64,17 +72,26 @@ if (!(await createTokenBridgeEnoughCustomFeeTokenAllowance(allowanceParams))) {
...
 
-The deployment of token bridge contracts constitutes the foundational step in establishing a bridge between the parent and the Orbit chain. This process mirrors the deployment methodology used for orbit chain contracts, where a primary contract, named `RollupCreator`, facilitates the deployment of core contracts. In the context of token bridge contracts, the `TokenBridgeCreator` contract assumes a similar pivotal role by orchestrating the deployment across both the parent and the Orbit chain. Feel free to take a look at the [`TokenBridgeCreator` Solidity code](https://github.com/OffchainLabs/token-bridge-contracts/blob/b3894ecc8b6185b2d505c71c9a7851725f53df15/contracts/tokenbridge/ethereum/L1AtomicTokenBridgeCreator.sol#L4).
+The deployment process is the same as orbit chain contracts', where a primary contract facilitates the deployment of core contracts. The token bridge contracts are deployed on the parent and child chains by `TokenBridgeCreator`. `TokenBridgeCreator` does it in a single transaction using the [ Retryable Tickets protocol ](/arbos/l1-to-l2-messaging#retryable-ticketsO).
```
Comment (symbolpunk) [inline arbitrum-docs/launch-orbit-chain/how-tos/orbit-sdk-deploying-token-bridge.md:90]:
```suggestion
The deployment process is the same as Orbit chain contracts', where a primary contract facilitates the deployment of core contracts. The token bridge contracts are deployed on the parent and child chains by `TokenBridgeCreator`. `TokenBridgeCreator` does it in a single transaction using the [ Retryable Tickets protocol ](/arbos/l1-to-l2-messaging#retryable-ticketsO).
```

Could possibly search for `orbit` and ensure they're all corrected to `Orbit`

**Review by symbolpunk — APPROVED** [review]
Approving to unblock merging, with some nits/suggestions left in comments above 👍

**Pull request merged**